stat/roc.go

```diff
@@ -127,3 +129,67 @@ func ROC(cutoffs, y []float64, classes []bool, weights []float64) (tpr, fpr, thr
 
 	return tpr, fpr, cutoffs
 }
+
+// TOC returns the Total Operating Characteristic for the classes provided
+// and the minimum and maximum bounds for the TOC.
+//
+// The input classes should be sorted with their weights according to the
+// rank variable ascending. SortWeightedLabeled can be used to sort classes
```

Comment:
This is confusing for me. The rank variable itself relates to the order of data in the sample and where a particular value fits in it. I think it would be *much* clearer for the user if we just said sth like "y values must be sorted in ascending order, classes[i] is the class of value y[i] and weights[i] is the weight of y[i]", even at the cost of some verbosity.

Reply:
Done.